### DISCUSSION TITLE
convert-dlrs-rules.apex script completes successfully, but no CMDT created

### DISCUSSION THREAD

**Discussion by jdids247** [jamessimone/apex-rollup#501] [Q&A] [answered]
Hopefully this is a simple issue.

When I run this script (convert-dlrs-rules.apex) from the Developer Console using the Debug > Open Execute Anonymous option.

Everything runs and the logs show it was successful, but there are no CDMT records created.

We're in a developer org testing currently. I'm setup as a System Administrator. Hoping I'm missing something painfully obvious here :)

**Comment by jamessimone** [comment]
Have you looked through the log that's generated when you run that script? You should be able to open it from the Log panel in the Developer Console. If you enable the "Debug Only" checkbox, you should see some more info about what's going on.

**Reply by jdids247** [reply]
Checking the "Debug Only" box returned this. Still says it completed successfully

![image](https://github.com/jamessimone/apex-rollup/assets/110419431/d77fbf93-d1eb-40d6-8bb4-728b13824854)

I'm also attaching the log
[apex-07LE10000033IDBMA2.log](https://github.com/jamessimone/apex-rollup/files/12704479/apex-07LE10000033IDBMA2.log)

**Reply by jamessimone** [reply, marked as answer]
From that screenshot, it seems a deployment to create the custom metadata records has been queued up -- I would check the Deployment Status page in setup for more details

**Reply by jdids247** [reply]
Thanks James, you were spot on. Looks like your solution doesn't let you use the "Case Comment" object, which was causing the deployment issue.

**Reply by jamessimone** [reply]
Ah yeah. Some objects aren't supported by custom metadata, but you can easily recreate the functionality using Flow for those objects with the invocable, or (if you're comfortable with Apex), you can do this in Apex as well. I have a portion of one of my setup videos showing this off: https://www.youtube.com/watch?v=RyQHXi5boW0&t=528s
